gradle.properties

```diff
@@ -1,3 +1,3 @@
 grailsVersion=7.0.0
```

Comment:
Is this `grailsVersion` property used anywhere? I does not seem so. And it is invalid, right?